src/main/java/org/jvnet/hudson/plugins/seleniumhtmlreport/SeleniumHtmlReportPublisher.java

```diff
@@ -69,16 +67,16 @@ public BuildStepMonitor getRequiredMonitorService() {
     }
 
     @Override
-    public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
+    public void perform(Run<?, ?> build, FilePath workspace, Launcher launcher, TaskListener listener) throws InterruptedException, IOException {
         listener.getLogger().println("Publishing Selenium report...");
-        FilePath seleniumResults = build.getWorkspace().child(this.testResultsDir);
+        FilePath seleniumResults = workspace.child(this.testResultsDir);
         if (!seleniumResults.exists()) {
             listener.getLogger().println("Missing directory " + this.testResultsDir);
-            return false;
+            return;
```

Comment:
I assume you want this to be a build failure, as I think returning false used to do, so

```java
throw new AbortException("Missing directory " + this.testResultsDir);
```

instead of printing a message and letting the build succeed.